### PR TITLE
PCSX2: Remove obsolete "DisableOutput" option from core/GUI

### PR DESCRIPTION
**Pull request by MrCK1**
It's been broken/unused for some time.

### CODE REVIEW

**Comment by gregory38** [comment]
```
It's been broken/unused for some time and is superseeded by the "Core Benchmark" option that exists internally, inside GSdx.
```
There aren't the same option.

This option removes all traffics between GSdx and the core (so basically MTGS is idling).
The GSdx option is a null renderer, so you still emulate GS register, texture transfers. To be fair, remaining activity should be light.

**Comment by MrCK1** [comment]
Ah OK, didn't realize they were actually different options, they seemed like the same thing. Guess I'll close this then :P

**Comment by gregory38** [comment]
Honestly if the core option is broken. I'm not sure it is worth to keep it.

**Comment by FlatOutPS2** [comment]
Is it really broken though?

**Comment by gregory38** [comment]
Maybe we could do a test to check if there is any real difference between the options.

**Comment by MrCK1** [comment]
I tried it earlier, the comment around the GUI option in Video panel says the same thing. 

Another caveat is that this option doesn't work on the fly like GS Null does, it reauires a restart of something (can't quite recall right now)

I'll have to try again in a few hours, but I didn't find anything of use personally.

**Comment by turtleli** [comment]
Seems to infinite loop in CopyGSPacketData() when the disable output option is used.

**Comment by MrCK1** [comment]
@turtleli Do you still want to keep this open? I can change the commit message if necessary.

**Comment by refractionpcsx2** [comment]
@turtleli

**Comment by turtleli** [comment]
Yeah, change the commit message so it doesn't mention the superseded stuff.

**Comment by FlatOutPS2** [comment]
These travis build failures not caused by coding mistakes have been quite common lately. I had one yesterday too. This PR even has 2 travis builds fail without finishing.

**Comment by lightningterror** [comment]
I'd still do a rebase until it compiles property.
Having big red X marks on merged PRs doesn't look good.

**Comment by FlatOutPS2** [comment]
You could add "It's been broken/unused for some time." to the commit message instead of the title to make the push worthwhile. :)

**Comment by turtleli** [comment]
I think you both have the correct permissions to restart the affected Travis CI jobs if they fail?

**Comment by lightningterror** [comment]
Yeah we can, that did the trick.

**Comment by turtleli** [comment]
It looks like you restarted the entire build. Only the individual jobs that failed need to be restarted. It consumes less resources that way.

**Comment by lightningterror** [comment]
Yeah, I noticed that I can do that after I restarted the entire build.

File: pcsx2/Config.h [hunk 1]
```diff
@@ -318,7 +318,6 @@ struct Pcsx2Config
 		{
 			return
 				OpEqu( SynchronousMTGS )		&&
-				OpEqu( DisableOutput )			&&
```
Comment (turtleli) [inline pcsx2/Config.h]: I think you still need to remove the DisableOutput variable.

**Pull request merged**